PD-232   Rework Network Port ID Articles: R-Series (#1866)
truenas/documentation: content/Hardware/RSeries/RSeriesNetworkPortIDs.md

{{< toc >}}

TTrueNAS identifies the network ports on TrueNAS R-Series systems based on what software is installed.

{{< include file="/_includes/NetworkPortIdentifications.md" type="page" >}}

By default, a TrueNAS R-Series with CORE installed identifies its network ports with **ixl** followed by a number.

SCALE identifies its network ports with **eno** followed by a number.

> The tables within the expandable boxes below list the default identification for R-Series systems and any add-on networking cards that iXsystems has qualified for use with R-Series systems.

Reviewer: hum... maybe this?
```suggestion
The expandable boxes below contain tables that list the default identification for R-Series systems and any add-on networking cards qualified by iXsystems for use with R-Series systems.
```

Author: That sounds good.

## R10

> 

Reviewer: Why are you using the code emphasis for the port values? if not in a or as command, using the ` ` tags is not appropriate emphasis. You could add a column to these tables for Port number, then the value in another column but still not with the command emphasis.
